enhancements/console/dynamic-plugins.md

```diff
@@ -173,21 +184,22 @@ then passed in the HTTP `Authorization` request header, for example:
 apiVersion: console.openshift.io/v1alpha1
 kind: ConsolePlugin
 metadata:
-  name: my-plugin
+  name: acm-console-plugin
```

Comment:
I'd like to discourage plugin authors from putting `-plugin` at the end of all the plugin names :) Let's try to remove that from the examples.

```suggestion
  name: acm
```